pages/docs/installation/installation-special-systems.md

```diff
@@ -16,6 +16,55 @@ The instructions may still be valuable for unlisted systems.
+
+(1) [Download Eigen](http://eigen.tuxfamily.org/index.php?title=Main_Page) and copy it to HAWK. Afterwards export the `EIGEN3_ROOT`, e.g.,
+
+```bash
+export EIGEN3_ROOT="$HOME/precice/eigen"
+```
+
+(2) Load available modules:
+
+```bash
+module load cmake boost petsc/<VERSION>-int32-shared
+```
+
+(3) Build preCICE. For PETSc, the library and include path (as opposed to the directory and/or architecture, as usual) need to be defined explicitly:
+
+```bash
+cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX="my/install/prefix" -DPRECICE_PETScMapping=ON -DPETSc_INCLUDE_DIRS="$PETSC_DIR/include" -DPETSc_LIBRARIES="$PETSC_DIR/lib/libpetsc.so" -DPRECICE_PythonActions=OFF .. /path/to/precice/source
+
+make install -j 16
+```
+
+Usually, both variables, `PETSc_LIBRARIES` and `PETSc_INCLUDE_DIRS` are supposed to be found by `cmake`. This might be an issue with our PETSc detection mechanism or an issue with the cluster. If you find a more native way to use the provided PETSc installation, please update this documentation. The PETSc module, where this issue occurred, was `petsc/3.12.2-int32-shared`.
+
+{% include important.html content="Some tests (`acceleration`, `mpiports`, `serial` and `parallel`) are known to fail, although the installation is fine, since the tests require MPI features that are disabled by default (Spawn Capable MPI runs). If you know how to enable this feature, please edit the page here." %}
+
+#### Running on a single node
+
+Simulations on a single node are possible, but you explicitly need to specify the hardware. Otherwise, the MPI jobs are executed on the same cores, which will slow down the whole simulation due to migration significantly. In order to run the a coupled simulation on a single node, use the following:
+
+```bash
+mpirun -np 4 omplace -nt 1 ./exec1 args &
+mpirun -np 4 omplace -b 4 -nt 1 ./exec2 args2
+```
+
+The `nt` argument specifies the number of threads each rank uses. Since we don't want to use multi-threading, we select just a single thread per core. The argument option `-b` specifies the starting CPU number for the effective CPU list, so that we shift the starting number of CPU list in the second participant by the cores employed for the first participant. There are further options to specify the hardware. Have a look at the `omplace` using `man omplace` or the [hardware pinning](https://kb.hlrs.de/platforms/index.php/Batch_System_PBSPro_(Hawk)#Pinning) documentation for more information.
+
+#### Notes on deal.II
+
+A variety of dependencies build on the library `METIS`, which requires an installation of `LAPACK`. In order to use `LAPACK` you can load the module `libflame`. Assuming that the preCICE modules above are loaded, you can install deal.II as usual (metis will already be loaded due to a dependency of PETSc). Additional dependecies such as TRILINOS are available through the package manager and can be loaded as well.
```

Comment:
"A variety of dependencies" What does this mean? A variety of dependencies in general or dependencies of deal.II? I would like this to be more explicit. 

I find the information about `METIS` confusingly formulated. Is `METIS` preinstalled or not? You tell me to load some module for `LAPACK` because it is needed for the installation of `METIS`. On the other hand it sounds like there is `METIS` preinstalled. I just checked and `METIS` is indeed preinstalled. Maybe refer explicitly to some `METIS` module?! 

What does "are available through the package manager" mean? Which package manager? Do you mean the module system?

```suggestion
`METIS` is preinstalled and can be loaded via the module system. In case that the preCICE modules above are loaded, `METIS` will already be loaded as a dependency of PETSc. If you want to compile `METIS` yourself, you can load the module `libflame` to fulfill the `LAPACK` dependency. . In order to use `LAPACK` you can load the module `libflame`. 

Additional dependencies of deal.II, such as `TRILINOS`, are available through module system and can be loaded accordingly. You can get obtain the a full list of preinstalled software on Hawk using the command `module avail`.
```

Reply:
I tried to clarify it. `LAPACK` is here a requirement of deal.II, not of `METIS`